projects/hr-portal/src/hr-portal.ts

```diff
@@ -1,83 +1,265 @@
+  IgcButtonComponent,
+  IgcIconButtonComponent,
+  IgcIconComponent,
+  registerIcon,
+} from "igniteui-webcomponents";
+import {
+  IgcTreeGridComponent,
+  IgcCellTemplateContext,
+} from "igniteui-webcomponents-grids/grids";
+import { icons } from "./assets/icons/Icons";
+import { DataService } from "./services/data.service";
+
+defineComponents(
+  IgcAvatarComponent,
+  IgcIconComponent,
+  IgcTreeGridComponent,
+  IgcIconButtonComponent,
+  IgcButtonComponent
+);
+configureTheme("fluent");
 
 @customElement("app-hr-portal")
 export default class HrPortal extends LitElement {
-  render() {
-    return html` <div>HR PORTAL</div> `;
+  @property({ type: Array })
+  data: Array<any> = [];
+
+  @property({ type: Boolean })
+  isSorted: boolean = false;
+
+  private dataService: DataService;
+
+  constructor() {
+    super();
+    this.data = [];
+    this.dataService = new DataService();
+    icons.forEach((icon: { name: string; path: string; category: string }) => {
+      registerIcon(icon.name, icon.path, icon.category);
+    });
   }
 
+  connectedCallback() {
+    super.connectedCallback();
+    this.dataService.fetchData().then((data) => {
+      this.data = data;
+    });
+  }
 
+  @query("igc-tree-grid")
+  private treeGrid!: IgcTreeGridComponent;
 
-  static styles = css`
-    :host {
-      max-width: 1280px;
-      margin: 0 auto;
-      padding: 2rem;
-      text-align: center;
+  clearSorting() {
+    if (this.treeGrid) {
+      this.treeGrid.sortingExpressions = [];
     }
+    this.isSorted = false;
+  }
 
-    .logo {
-      height: 6em;
-      padding: 1.5em;
-      will-change: filter;
-      transition: filter 300ms;
+  handleSortingChanged() {
+    if (this.treeGrid) {
+      this.isSorted = this.treeGrid.sortingExpressions.length > 0;
+      console.log(
+        "Sorting expressions changed:",
+        this.treeGrid.sortingExpressions
+      );
     }
-    .logo:hover {
-      filter: drop-shadow(0 0 2em #646cffaa);
-    }
-    .logo.lit:hover {
-      filter: drop-shadow(0 0 2em #325cffaa);
+  }
+
+  public avatarTemplate = (ctx: IgcCellTemplateContext) => {
+    let row = ctx.cell.row;
+    return html`
+      <div class="employeeDiv">
+        <igc-avatar shape="rounded" .src="${row.data.Picture}"></igc-avatar>
+        <span>${row.data.Name}</span>
+      </div>
+    `;
+  };
+
+  public countryIconTemplate = (ctx: IgcCellTemplateContext) => {
+    let row = ctx.cell.row;
+    return html` <div class="flagDiv">
+      <igc-icon
+        collection="country-icons"
+        name="${row.data.Country}"
+      ></igc-icon>
+      <span>${row.data.Location}, ${row.data.Country}</span>
+    </div>`;
+  };
+
+  public contactsTemplate = (ctx: IgcCellTemplateContext) => {
+    let row = ctx.cell.row;
+    return html`
+      <div class="center-content">
+        <a href="mailto:${row.data.Email}">
+          <igc-icon-button
+            collection="hr-icons"
+            name="mail"
+            class="small"
```

Comment:
You can move the class `small` to the parent div so that you dont apply the same class over again on all elements

Reply:
moved to the parent div